benchmark/benchmarks.jl

```diff
@@ -34,57 +42,64 @@ for subdir ∈ subdirs
-      push!(ratio_symrcm, nnz(ldlt_symrcm) / nnz_A)
-      push!(ratio_classic, nnz(ldlt_classic) / nnz_A)
+      SUITE["no_ordering"][name] = @benchmarkable nnz(ldl($A, $p_classic).L)/$nnz_A
+      SUITE["amd"][name] =  @benchmarkable nnz(ldl($A, $p_amd).L)/$nnz_A
+      SUITE["symamd"][name] = @benchmarkable nnz(ldl($A, $p_symamd).L)/$nnz_A
+      SUITE["metis"][name] = @benchmarkable nnz(ldl($A, $p_metis).L)/$nnz_A
+      SUITE["symrcm"][name] = @benchmarkable nnz(ldl($A, $p_symrcm).L)/$nnz_A
     end
   end
 end
 
-orderings = [:NO_ORDERING, :AMD, :SYMAMD, :METIS, :SYMRCM]
-n = length(names_sqd)
-stats = Dict(ordering =>
-          DataFrame(
-            :id     => 1:n,
-            :name   => [@sprintf("%15s", names_sqd[i]) for i = 1:n],
-            :nnz    => [@sprintf("%6d", nnz_sqd[i]) for i = 1:n],
-            :ratio  => [@sprintf("%3.1f", (ordering == :AMD ? ratio_amd[i] : (ordering == :SYMAMD ? ratio_symamd[i] : (ordering == :METIS ? ratio_metis[i] : (ordering == :SYMRCM ? ratio_symrcm[i] : ratio_classic[i]))))) for i = 1:n]
-          ) for ordering in orderings)
+# orderings = [:NO_ORDERING, :AMD, :SYMAMD, :METIS, :SYMRCM]
+# n = length(names_sqd)
+# stats = Dict(ordering =>
+#           DataFrame(
+#             :id     => 1:n,
+#             :name   => [@sprintf("%15s", names_sqd[i]) for i = 1:n],
+#             :nnz    => [@sprintf("%6d", nnz_sqd[i]) for i = 1:n],
+#             :ratio  => [@sprintf("%3.1f", (ordering == :AMD ? ratio_amd[i] : (ordering == :SYMAMD ? ratio_symamd[i] : (ordering == :METIS ? ratio_metis[i] : (ordering == :SYMRCM ? ratio_symrcm[i] : ratio_classic[i]))))) for i = 1:n]
+#           ) for ordering in orderings)
 
-df = join(stats, [:ratio], invariant_cols=[:name, :nnz])
-hdr_override = Dict(:ratio_NO_ORDERING => "NO\\_ORDERING", :ratio_METIS => "METIS", :ratio_AMD => "AMD", :ratio_SYMRCM => "SYMRCM", :ratio_SYMAMD => "SYMAMD")
+# df = join(stats, [:ratio], invariant_cols=[:name, :nnz])
+# hdr_override = Dict(:ratio_NO_ORDERING => "NO\\_ORDERING", :ratio_METIS => "METIS", :ratio_AMD => "AMD", :ratio_SYMRCM => "SYMRCM", :ratio_SYMAMD => "SYMAMD")
 
-open("benchmarks.md", "w") do io
-  pretty_stats(io, df, hdr_override=hdr_override)
-end
+# open("benchmarks.md", "w") do io
+#   pretty_stats(io, df, hdr_override=hdr_override)
+# end
 
-if latex_benchmarks
-  open("benchmarks.tex", "w") do io
-    println(io, "\\documentclass[varwidth=20cm,crop=true]{standalone}")
-    println(io, "\\usepackage{longtable}")
-    println(io, "\\begin{document}")
-    pretty_latex_stats(io, df, hdr_override=hdr_override)
-    println(io, "\\end{document}")
-  end
-end
+# if latex_benchmarks
+#   open("benchmarks.tex", "w") do io
+#     println(io, "\\documentclass[varwidth=20cm,crop=true]{standalone}")
+#     println(io, "\\usepackage{longtable}")
+#     println(io, "\\begin{document}")
+#     pretty_latex_stats(io, df, hdr_override=hdr_override)
+#     println(io, "\\end{document}")
+#   end
+# end
```

Comment:
I think this can all be removed too. It's easy to output LaTeX with SolverBenchmarks. That will simplify your script a lot.